icu4j/maven-migration/toMaven.sh

```diff
@@ -36,6 +47,13 @@ function rmFileIfExists() {
   [ -f "$1" ] && rm "$1"
 }
 
+function copyRecursive() {
+  SRC=$1
+  DEST=$2
+
+  cp -R "$1" "$2"
```

Comment:
Same, no point in defining them.
BUT!
If you choose to define (and use) them, then you can do 
```
local SRC=...
local DST=...
```

Reply:
Done.